app/views/projects/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1><%= @project.name %></h1>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
```

Comment:
Doing a `count` in the view is considered an MVC infraction because we are now making a call directly to the database. While it may seem that we are using `count` from pure Ruby, we are actually using AR here.

We want to put this in the model so that we can test it.